### PR TITLE
chore: standardise .gitignore to ignore IDE and macOS files

### PR DESCRIPTION
**Pull request by dan2k3k4**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,19 @@
+# Node
 node_modules
 .npmrc
 
-# Local Netlify folder
-.netlify
```
Comment (Leksat) [inline .gitignore:6]: Are you sure about this removal?

Comment (dan2k3k4) [reply .gitignore:6]: Hmm that was supposed to stay 🙈

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,23 @@
+# Node
 node_modules
 .npmrc
 
+# JetBrains
+**/.idea/workspace.xml
+**/.idea/tasks.xml
+**/.idea/shelf
```
Comment (Leksat) [inline .gitignore]: PhpStorm config is already under git: https://github.com/AmazeeLabs/silverback-mono/blob/development/.idea/.gitignore

So I guess we don't need this.

**Review by Leksat — CHANGES_REQUESTED** [review]

**Comment by Leksat** [comment]
@dan2k3k4 Once you have everything done, could you please squash all commits?

File: .gitignore [hunk 3]
```diff
@@ -1,8 +1,13 @@
 _local
-node_modules
 .phpunit.result.cache
 
 # Local Netlify folder
 .netlify
 
+# Node / Deno
+node_modules
+.npmrc
```
Comment (Leksat) [inline .gitignore]: Why to ignore this one?

Comment (dan2k3k4) [reply .gitignore]: Hmm, so the `Gitignore` changes came from using `npx gitignore`

I see that we already have a `.npmrc` committed, and we're not really using `npm` anywhere. I don't think `pnpm` reads the `.npmrc` file.

So I've removed it from the branch

Comment (Leksat) [reply .gitignore]: https://pnpm.io/8.x/npmrc 😉

**Review by Leksat — APPROVED** [review]

**Pull request merged**